nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/cf/JMSConnectionFactoryProvider.java

```diff
@@ -257,21 +256,26 @@ private void setConnectionFactoryProperties(ConfigurationContext context) {
      */
     private void setProperty(String propertyName, Object propertyValue) {
         String methodName = this.toMethodName(propertyName);
-        Method method = Utils.findMethod(methodName, this.connectionFactory.getClass());
-        if (method != null) {
+        Method[] methods = Utils.findMethods(methodName, this.connectionFactory.getClass());
+        if (methods != null && methods.length < 0) {
```

Comment:
methods.length can never be less than 0 here... was that supposed to be > 0 perhaps?

Reply:
Yes sorry, it was supposed to be methods.length > 0
I will change and open a new pull request